app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :users
+  belongs_to :posts
+
+  def update_likes_counter
+    post.increment(:likes_counter)
+  end
+end
```

Comment:
When creating a new like, I get the following error: 
![image](https://github.com/ShiroYaksha90/blog-app/assets/93520178/50a5b7b0-5a76-4098-8319-9d73edd0e1c9)
This is probably because of the way you named the post and the user when making the association.
I suggest you replace line 2 and 3 with the following: 
```
belongs_to :author, class_name: 'User'
belongs_to :post
```

[OPTIONAL] It would be great if you used `after_save` to call the #update_likes_counter method every time a new like has been saved to database, then make the `update_like_counter` method private.